### PR TITLE
EVA-3668: Clarify support for compressed input files

### PR DESCRIPTION
**Pull request by apriltuesday**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,33 +48,39 @@ Install each of these and ensure they are included in your PATH. Then install th
...
-## eva-sub-cli tool: Options and parameters guide
+## Options and parameters guide
 
-The eva-sub-cli tool provides several options/parameters that you can use to tailor its functionality to your needs. Understanding these parameters is crucial for configuring the tool correctly. Below is an overview of the key parameters and options:
+The eva-sub-cli tool provides several options and parameters that you can use to tailor its functionality to your needs.
+You can view all the available parameters with the command `eva-sub-cli.py -h` and view detailed explanations for the
+input file requirements in the ["Getting Started" guide](Getting_Started_with_eva_sub_cli.md).
+Below is an overview of the key parameters.
 
-| OPTIONS/PARAMETERS         | DESCRIPTION                                                                                                                                                                                                        |
```
Comment (apriltuesday) [inline README.md:55]: We can discuss if people think we should keep this table here as well as in the `help` command, if we do we just need to remember to keep it in sync.

I've tried to make the documentation here more of a "quick reference" and left the more detailed explanation and troubleshooting guide to the "Getting started" document. Let me know what you think about this distinction.

File: eva_sub_cli/executables/xlsx2json.py [hunk 2]
```diff
@@ -47,6 +47,7 @@ def __init__(self, xlsx_filename, conf_filename):
         :param conf_filename: configuration file path
         :type conf_filename: basestring
         """
+        self.errors = []
```
Comment (apriltuesday) [inline eva_sub_cli/executables/xlsx2json.py:50]: (this is unrelated but fixes an issue that came up during testing)

**Review by tcezard — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -48,33 +48,39 @@ Install each of these and ensure they are included in your PATH. Then install th
...
 
 The metadata can also be provided via a JSON file, which should conform to the schema located [here](eva_sub_cli/etc/eva_schema.json).
+This is passed using the option `--metadata_json`.
+
+### VCF files and Reference FASTA
+
+These can be provided either in the metadata file directly, or on the command line using the `--vcf_files` and `--reference_fata` options.
+Note that if you are using more than one reference FASTA, you **cannot** use the command line options; you must specify which VCF files use which FASTA files in the metadata.
 
+VCF files can be either uncompressed or compressed (bgzipped). FASTA files must be uncompressed.
```
Comment (Dona094) [inline README.md:83]: Here should we also specify that if the files are not bgzipped and with other compression you are likely to get an error?

Comment (apriltuesday) [reply README.md]: good point, added!

File: README.md [hunk 4]
```diff
@@ -48,33 +48,41 @@ Install each of these and ensure they are included in your PATH. Then install th
...
-| --shallow                  | Set the validation to be performed on the first 10000 records of the VCF. Only applies if the number of record exceed 10000                                                                                        |
-| --username                 | Username used for connecting to the ENA webin account                                                                                                                                                              |
-| --password                 | Password used for connecting to the ENA webin account                                                                                                                                                              |
+### Submission directory
 
+This is the directory where all processing will take place, and where configuration and reports will be saved.
+Crucially, the eva-sub-cli tool requires that there be **only one submission per directory**.
+Running multiple submissions from a single directory can result in data loss during validation and submission.
```
Comment (apriltuesday) [inline README.md:62]: @tcezard Based on the discussion this morning, do you think this is clear enough? I was going to add something about not reusing a directory, but I think that's the same thing as running multiple from a single directory..

Comment (apriltuesday) [reply README.md:62]: updated to clarify reusing is also not allowed

Comment (tcezard) [reply README.md:62]: Makes sense

**Pull request merged**